clients/drcachesim/common/options.cpp

```diff
@@ -117,6 +117,14 @@ droption_t<std::string> op_alt_module_dir(
     "analysis tools, or in the raw modules file for post-prcoessing of offline "
     "raw trace files.  This directory takes precedence over the recorded path.");
 
+droption_t<bool> op_ignore_decode_failure(
+    DROPTION_SCOPE_FRONTEND, "ignore_decode_failure", false,
+    "Whether instruction decode failures should be ignored",
+    "Specifies whether failures to decode the instruction encodings should be ignored by "
+    "the analysis tools. This is useful especially when decoding kernel traces collected "
+    "by Intel-PT where decode failures may not indicate a critical issue and can be "
+    "moved past.");
```

Comment:
I looked into pt2ir.cpp and I see this:
```
            /* TODO i#2103: Currently, the PT raw data may contain 'STAC' and 'CLAC'
             * instructions that are not supported by Dynamorio.
             */
```
If those are the only 2 missing opcodes, let's just add them to the DR decoder and then not need any of these options.  I suspect it will not be much more work to add those opcodes than to add and plumb through all these options.

Reply:
Done. #6484